using-mobile-extensions/mobile-core/identity/README.md

```diff
@@ -49,31 +49,31 @@ import ACPCore
 {% endtab %}
 
 {% tab title="React Native" %}
-#### JavaScript
+### JavaScript
 
 Import the Identity extension
 
 ```jsx
 import {ACPIdentity} from '@adobe/react-native-acpcore';
 ```
 
-Get the extension version
-
-```jsx
-ACPIdentity.extensionVersion().then(version => console.log("AdobeExperienceSDK: ACPIdentity version: " + version));
-```
 {% endtab %}
 {% endtabs %}
 
-## **Register the extension**
+
+
+## **Register the Identity extension**
```

Comment:
I don't think we need the asterisks when the text has been tagged as a heading, so please remove the asterisks.